extra_scripts/fw-version.py

```diff
@@ -0,0 +1,8 @@
+import subprocess
```

Comment:
replace via https://community.platformio.org/t/how-to-build-got-revision-into-binary-for-version-output/15380/4 ?